src/evict/evict_lru.c

```diff
@@ -168,29 +171,27 @@ __wt_evict_server_wake(WT_SESSION_IMPL *session)
 }
 
 /*
- * __evict_thread_run --
- *	General wrapper for any eviction thread.
+ * __wt_evict_worker_run --
+ *	General wrapper for an eviction worker thread.
  */
-static WT_THREAD_RET
-__evict_thread_run(void *arg)
+int
+__wt_evict_worker_run(WT_SESSION_IMPL *session, WT_WORKER_THREAD *worker)
```

Comment:
To me, `worker` sounds subordinate, implying some other thread is more special or controlling things.  Whereas the old name `thread` made it sound like all threads are equal.  If all eviction threads are using it we should retain the name, adding the prefix, and keep the comment to make it clear any/all eviction threads are in here.